### DISCUSSION TITLE
Contour Content-Encoding header

### DISCUSSION THREAD

**Discussion by VA-PLE** [projectcontour/contour#5325] [Q&A]
Hi. A http client sends a request encoded with gzip (using a Content-Encoding header), can Contour decrypt that transparently and then send the decoded body to the service?

**Comment by skriss** [comment]
Not currently, Contour does not have a way to enable the [decompressor filter](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/decompressor_filter). Please file a feature request with some additional details on your use case if this is something you are interested in!
